File: Learn the Basic/Pattern_printing/Alpha_Ramp_Pattern.py
def Alpha_Ramp_Pattern(n):
    ch=ord('A')
    for  i in range(1,n+1):
        for j in range(1,i+1):
            print(chr(ch),end="")
        print()
        ch=ch+1
n=int(input("Enter the value of n: "))
Alpha_Ramp_Pattern(n)

File: Learn the Basic/Pattern_printing/Alpha_Ramp_Pattern.cpp
#include<iostream>
using namespace std;
void Alpha_Ramp_Pattern(int n){
    char ch='A';
    for(int i=1;i<=n;i++){
        for(int j=1;j<=i;j++){
            cout<<ch;
        }
        cout<<endl;
        ch=ch+1;
    }
}

int main(){
    int n;
    cout<<"Enter the value of n: ";
    cin>>n;
    Alpha_Ramp_Pattern(n);
    return 0;
}